app/src/main/res/layout-land/activity_movie_details.xml

```diff
@@ -52,27 +51,29 @@
 
             <TextView
                 android:id="@+id/tv_title"
-                android:layout_width="match_parent"
+                android:layout_width="200dp"
                 android:layout_height="wrap_content"
                 android:layout_marginStart="18dp"
                 android:layout_marginTop="10dp"
-                android:layout_marginEnd="200dp"
+                android:layout_marginEnd="160dp"
                 android:text="@string/title"
                 android:textColor="@color/whiteColor"
                 android:textSize="40sp"
```

Comment:
All dimensions value sp, dp, numbers you must be extracted as resources.
How variant you can use this principle
https://medium.com/mindorks/android-resource-naming-convention-42e4e8026614